src/themes/colors.ts

```diff
@@ -8,6 +8,8 @@ enum Colors {
   PrimaryText = '#FFF',
   SecondaryText = '#8A8A8A',
   Dev = '#FF0000',
+  BlackAlpha = 'rgba(0,0,0,1)',
+  Transparent = 'rgba(0,0,0,0)',
```

Comment:
[non-blocking] [nit] I was thinking about this and wondering, the rest of the names are quite semantic, wheras these colours are in a different form and the keys refer to their actual colour value. I wonder if it's better to just directly inline these in `gradient`?